components/02-content/shopping-cart/shopping-cart.hbs

```diff
@@ -3,32 +3,31 @@
 <div class="dso-shopping-cart">
   <div class="dso-contents">
     {{#if items }}
-      {{#if toggleButton }}
+      {{#if toggleButton ~}}
```

Comment:
wat doet die `~`

Reply:
http://handlebarsjs.com/expressions.html#whitespace-control

Reply:
Ah dat moet misschien dan nog wel veel vaker? Ik zie in de reference/render/ voorbeelden veel whitespace. Weet niet of het erg is hoor.

Reply:
Het is zeker waar dat significant whitespace te lang niet erkend of een ondergeschoven kindje geweest, maar waar jij het over hebt is een ander geval waar ik in de betreffende reactie op reageer